src/quo2/components/browser/browser_input/component_spec.cljs

```diff
@@ -0,0 +1,57 @@
+(ns quo2.components.browser.browser-input.component-spec
+  (:require [quo2.components.browser.browser-input.view :as browser-input]
+            [test-helpers.component :as h]))
+
+(h/describe "Browser input"
+  (h/test "Renders empty in default state"
+    (h/render [browser-input/browser-input
+               {:on-change-text (h/mock-fn)
+                :value          ""}])
+    (h/is-truthy (h/get-by-label-text :browser-input)))
+
+  (h/test "Value updates on change"
```

Comment:
nit: test name is not accurate. It tests the on change was called not that the value was changed.